### PR TITLE
Added the possibility to load a single amber out file to check just the timing

### PR DESCRIPTION
**Pull request by DrDomenicoMarson**
Added the possibility to load a single amber out file to check just the timing

### CODE REVIEW

**Comment by xiki-tempula** [comment]
Why do you need to do this in alchemtest? You are not adding any new files as far as I can see?

**Comment by DrDomenicoMarson** [comment]
> Why do you need to do this in alchemtest? You are not adding any new files as far as I can see?
I added two functions needed to load single files, which are the used in the new ```test_amber.py``` in ```alchemlyb```

```load_single_u_nk_file```
```load_single_dHdl_file```

I thought this was consistent with how the other tests inside ```test_amber.py``` worked. Also, I don't really know how I can load a file directly from ```test_amber.py``` :smile:

File: src/alchemtest/amber/access.py [hunk 1]
```diff
@@ -106,3 +106,19 @@ def load_invalidfiles():
...
+
+    return glob(join(module_path, 'simplesolvated/charge/0.00/ti-*.out'))
+
+
+def load_single_u_nk_file():
+    """Load a single dHdl file."""
+
+    module_path = dirname(__file__)
+
+    return glob(join(module_path, 'bace_CAT-13d~CAT-17a/solvated/vdw/0.0/ti-*.out.bz2'))
```
Comment (orbeckst) [inline src/alchemtest/amber/access.py:124]: You can achieve loading a single file with
```python
import alchemtest.amber
bace = alchemtest.amber.load_bace_example()

single_u_nk = bace.data['solvated']['vdw'][0]
```

File: src/alchemtest/amber/access.py [hunk 2]
```diff
@@ -106,3 +106,19 @@ def load_invalidfiles():
...
     return Bunch(data=data,
                  DESCR=fdescr)
+
+
+def load_single_dHdl_file():
+    """Load a single dHdl file."""
+
+    module_path = dirname(__file__)
+
+    return glob(join(module_path, 'simplesolvated/charge/0.00/ti-*.out'))
```
Comment (orbeckst) [inline src/alchemtest/amber/access.py:116]: You can achieve loading a single file with
```python
import alchemtest.amber
solvated = alchemtest.amber.load_simplesolvated()

single_dHdl = solvated.data['charge'][0]
```

**Review by orbeckst — CHANGES_REQUESTED** [review]
I don't think the additional functions are necessary because you can pull out filenames from the `data` dict of the bunch that you get from the existing loaders. It's just about figuring out how the dicts are organized. See comments, please.

**Comment by xiki-tempula** [comment]
I echo @orbeckst 's point, I think these tests could be easily made in the alchemlyb tests and we only want alchemtest to be storing the data.

File: src/alchemtest/amber/access.py (hunk 1 above)
Comment (DrDomenicoMarson) [reply src/alchemtest/amber/access.py:124]: Thank you, I completely missed that, I fixed it!

File: src/alchemtest/amber/access.py (hunk 2 above)
Comment (DrDomenicoMarson) [reply src/alchemtest/amber/access.py:116]: Thank you, I completely missed that, I fixed it!

**Comment by DrDomenicoMarson** [comment]
Thank you for your support, I fixed my alchemlyb commit not to rely on the changes on alchemtest, so this PR can be safely closed and forgotten, I think! Can I close it or should some of you do that?

**Pull request closed without merging**